tests/expected/dry-run/expected

```diff
@@ -1,4 +1,6 @@
 kani-rustc --goto-c
 symtab2gb
-goto-cc --function main
+goto-cc
```

Comment:
I think it's vaguely questionable what exactly this test is looking for. We might consider retiring it later for the "complete exact output" UI kind of test so we can just bless the changes instead of what it's currently doing.